modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -613,10 +716,12 @@ PERF_TEST_P_(Cart2PolarPerfTest, TestPerformance)
 
 PERF_TEST_P_(CmpPerfTest, TestPerformance)
 {
-    CmpTypes opType = get<0>(GetParam());
-    cv::Size sz = get<1>(GetParam());
-    MatType type = get<2>(GetParam());
-    cv::GCompileArgs compile_args = get<3>(GetParam());
+    compare_f cmpF;
+    CmpTypes opType = CMP_EQ;
+    cv::Size sz;
+    MatType type = -1;
+    cv::GCompileArgs compile_args;
+    std::tie(cmpF, opType, sz, type, compile_args) = GetParam();;
```

Comment:
Please remove redundant `;` at the end of line.

Reply:
Applied